scripts/loadEmptyPrefsSets.js

```diff
@@ -0,0 +1,127 @@
+/*!
+Copyright 2018 Raising the Floor US
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+// This script generates unique pref sets and loads them into CouchDB.
+//
+// To create 20 empty prefs sets, the sctipt needs to be called like this:
+//   * COUCHDB_URL=http://localhost:8058/gpii NEW_USERS=20 node scripts/loadEmptyPrefsSets.js
+//
+// The options to this script must be passed as environment variables, and they are:
+//   * COUCHDB_URL: This neeeds to be in the form "http://host/dbname", but in case
+//   you need to provide the credentials to CouchDB, you need to use the form
+//   "http://user:pass@host:port/dbname".
+//   * NEW_USERS: The amount of new empty preferences sets you want to create
```

Comment:
Might be less confusing to use "keys" rather than "users" in this script.

Can you also mark each option with "required" or "optional". Thanks.

Reply:
> Might be less confusing to use "keys" rather than "users" in this script.

Okay, got a question about terminology. What I'm calling here "user" is represented by 2 tiers, a "prefSafe" and a "gpiiKey", right?. Is "key" the right term to use when we are referring to this? Sorry, I just want to understand and use the right terminology.

Reply:
Good point. Now I'm also not sure which term should be used. ;) @amb26, what do you think?